src/pages/billing/Project.js

```diff
@@ -81,23 +149,26 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
 
 
   // Render
-  const adminCanEdit = _.filter(({ roles }) => _.includes('Owner', roles), projectUsers).length > 1
-  const { displayName = null } = _.find({ accountName: billingAccountName }, billingAccounts) || {}
+  const adminCanEdit = _.filter(({ roles }) => _.includes(billingRoles.owner, roles), projectUsers).length > 1
+  const { displayName = null } = _.find({ accountName: billingAccountName }, billingAccounts) || { displayName: 'No Access' }
```

Comment:
Why not just change the default?
```js
const { displayName = 'No Access' } = _.find({ accountName: billingAccountName }, billingAccounts) || {}
```

Reply:
To me, this felt semantically more correct, because it is the name being displayed? (Also, the logic gets more complex down below, because we want the `Billing Account` prefix to show up regardless, so this folds that logic in cleanly.